types/uid-generator/uid-generator-tests.ts

```diff
@@ -1,14 +1,17 @@
-import { UIDGenerator } from 'uid-generator';
+import { default as UIDGenerator } from 'uid-generator';
 
-new UIDGenerator('abc'); // $ExpectType UIDGenerator
-const generator = new UIDGenerator(128, 'abc'); // $ExpectType UIDGenerator
+new UIDGenerator('abc'); // $ExpectType UIDGeneratorInstance
+const generator = new UIDGenerator(128, 'abc'); // $ExpectType UIDGeneratorInstance
 
 generator.generateSync(); // $ExpectType string
-generator.generate(uid => {
-  uid; // $ExpectType string
+generator.generate((err, uid?) => {
+  err; // $ExpectType Error | null
+  uid; // $ExpectType string | undefined
```

Comment:
```suggestion
  uid; // $ExpectType string
```
`uid` should, therefore, remain as a `string`

Reply:
This is the same as <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/39672#discussion_r340930766>, if some error triggered, the parameter uid could be `undefined`.